pom.xml

```diff
@@ -304,6 +304,26 @@
                     <version>${maven-coveralls-plugin.version}</version>
                 </plugin>
 
+
+                <plugin>
+                    <groupId>org.codehaus.mojo</groupId>
+                    <artifactId>findbugs-maven-plugin</artifactId>
+                    <version>3.0.1</version>
```

Comment:
Create property for version:
        <findbugs-maven-plugin>3.0.1</findbugs-maven-plugin>
and add the version here:
        <version>${findbugs-maven-plugin.version}</version>